app/views/sandbox/budget_guide.html.erb

```diff
@@ -0,0 +1,269 @@
+<div class="column">
+
+  <main class="content">
```

Comment:
No, but I need to have a body class to address this one as the margin is the header.